examples/deep_dream.py

```diff
@@ -162,16 +162,16 @@ def eval_loss_and_grads(x):
         grad_values = np.array(outs[1:]).flatten().astype('float64')
     return loss_value, grad_values
 
-# this Evaluator class makes it possible
-# to compute loss and gradients in one pass
-# while retrieving them via two separate functions,
-# "loss" and "grads". This is done because scipy.optimize
-# requires separate functions for loss and gradients,
-# but computing them separately would be inefficient.
-
 
 class Evaluator(object):
-
+    """
+    this Evaluator class makes it possible
```

Comment:
If making this a docstring, please format it as a proper docstring (see examples elsewhere in the codebase).